moban/main.py

```diff
@@ -27,6 +27,9 @@ def main():
     """
     parser = create_parser()
     options = vars(parser.parse_args())
+    if options[constants.LABEL_VERSION]:
```

Comment:
please declare LABEL_VERSION IN constants

Reply:
Or import `__version__` from `version.py` and use that.